doc/MANUAL.md

```diff
@@ -317,6 +326,50 @@ Windows has `ar` and `Expand` installed in default, but `7z` in particular might
 If you wan to run our test-suite or scan a zstd compressed file, We recommend installing this [7-zip-zstd](https://github.com/mcmilk/7-Zip-zstd)
 fork of 7zip. We are currently using `7z` for extracting `jar`, `apk`, `msi`, `exe` and `rpm` files.
 
+## Deployment best practices
+
+CVE binary tool was expected to be used by developers wishing to track known vulnerabilities against their own code and their dependencies.  We expect many users to use continuous integration systems such as Github Actions, and we have our own [cve-bin-tool GitHub Action](https://github.com/intel/cve-bin-tool-action) available to make that easy.  You can also run it within your existing build/test environments.  
```

Comment:
'was expected' or 'is expected'?

Reply:
I think I was thinking that once we released it I had to let go of my expectations of how people were going to use it, but it's probably clearer to say "is" in both cases anyhow.  Maybe I'll just change it to "is intended" instead of "expected"